README.md

```diff
@@ -66,13 +66,13 @@ sudo cp ~/Downloads/SidecarCore /System/Library/PrivateFrameworks/SidecarCore.fr
 8. Sign the patched SidecarCore (in Terminal):
     * If you see an error in this step, make sure you have xcode command-line tools installed (`xcode-select --install`, see [#3]) or updated (through App Store, see [#2]).
     * Don't restart your computer until you complete this step properly (or revert the backup file)! Many people have run into issues with this ([#28], [#22]).
-    * **macOS Catalina 10.15.4+ users: [Add `amfi_get_out_of_my_way=0x1` to NVRAM boot flags](https://github.com/ben-z/free-sidecar/issues/59#issuecomment-603953953), then skip step 9.**
+    * **macOS Catalina 10.15.4+ users:** Run [`sudo nvram boot-args="amfi_get_out_of_my_way=0x1"`](https://github.com/ben-z/free-sidecar/issues/59#issuecomment-603953953), **then skip step 9.**
```

Comment:
1. I had to scroll for a bit the linked thread in order to find the command I had to run, I think having the command here and keeping the link might be a best of both worlds
2. the whole sentence was bold but I think skipping step 9 deserves special attention